tests/ci/integration/run_openvpn_integration.sh

```diff
@@ -41,6 +41,7 @@ function openvpn_build() {
 
   OPENSSL_CFLAGS="-I/${AWS_LC_INSTALL_FOLDER}/include" \
   OPENSSL_LIBS="-L/${AWS_LC_INSTALL_FOLDER}/lib -lssl -lcrypto" \
+  LDFLAGS="-Wl,-rpath=/${AWS_LC_INSTALL_FOLDER}/lib" \
```

Comment:
Neat, this is a nice fix to avoid having to specify LD_LIBRARY_PATH in a lot of spots.